python/tests/test_pregenerated_models.py

```diff
@@ -22,8 +24,7 @@
                for case in list(expected_results[sub_test].keys())]
 
 
-@pytest.mark.skipif(os.environ.get('GITHUB_JOB') == 'valgrind',
-                    reason="Takes too long under valgrind")
+@skip_on_valgrind
```

Comment:
wasn't aware these tests are currently skipped and not sure whether it's a good idea since we lose quite a bit of coverage, especially regarding second order and event observables.

Reply:
Those are the same features that we are testing directly from C++ (also under valgrind). Unless we are worried about the swig-generated code, I don't think it adds much value to re-run those via Python. Even then, I don't think there are any swig-functions exercised in these tests that aren't already covered elsewhere.

Reply:
ah, good point!